src/Microsoft.Health.Fhir.SqlServer/Features/Storage/TvpRowGeneration/BulkTokenSearchParameterV1RowGenerator.cs

```diff
@@ -35,7 +35,7 @@ internal override bool TryGenerateRow(int offset, short searchParamId, TokenSear
             row = new BulkTokenSearchParamTableTypeV1Row(
                 offset,
                 searchParamId,
-                searchValue.System == null ? SqlSearchConstants.NullId : Model.GetSystemId(searchValue.System),
+                string.IsNullOrWhiteSpace(searchValue.System) ? SqlSearchConstants.NullId : Model.GetSystemId(searchValue.System),
```

Comment:
>IsNullOrWhiteSpace [](http://example.com/codeflow?start=23&length=18)

I would like to hear reasoning behind this case.

Reply:
I noticed for other rowGenerators like BulkQuantitySearchParameterV1RowGenerator, we are using string.IsNullOrWhiteSpace when retrieving the Id's from searchValue. The idea could be to avoid inserting white space.